src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx

```diff
@@ -241,6 +241,9 @@ Here are some possible fixes:
 * Make sure your browser agent meets these requirements:
   * Agent version v1.240.0 or greater. We recommend [upgrading to the latest agent version](/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes) as we continue making improvements.
   * Agent type: Pro or Pro+SPA. Session replay does not work with the Lite browser agent.
+  * Session Tracking and Session Trace features are enabled.
```

Comment:
@metal-messiah How do they enable these features? Are they available in the UI yet? I didn't see them in the main staging environment.